src/js/components/ServiceDetailDebugTab.js

```diff
@@ -0,0 +1,97 @@
+import moment from 'moment';
+import React from 'react';
+
+import DescriptionList from './DescriptionList';
+import Service from '../structs/Service';
+
+class ServiceDetailDebugTab extends React.Component {
+  getValueText(value) {
+    if (value == null || value === '') {
+      return (
+        <p>Unspecified</p>
+      );
+    }
+
+    return (
+      <span>{value}</span>
+    );
+  }
+
+  getLastTaskFailureInfo() {
+    let {lastTaskFailure} = this.props.service;
```

Comment:
Please use the [respective getter](https://github.com/dcos/dcos-ui/blob/master/src/js/structs/Service.js#L95)